lib/temporal/worker.rb

```diff
@@ -59,13 +59,15 @@ def start(reactor = nil)
     end
 
     def shutdown
+      core_worker.initiate_shutdown
```

Comment:
Can we hide `start` somehow and only have run? I have seen so many users use "start" and regret it because they can't catch a worker fatal exception. We have big regrets on it.

Also, can you confirm `run` does not return until shutdown is complete?

Reply:
`start` will be removed once I implement "run until" variant for run. I haven't forgotten, just sequencing the steps here for incremental progress.

And yes, `run` does not return until a shutdown is complete